react/AppLinker/native.js

```diff
@@ -1,14 +1,20 @@
-import { isAndroidApp } from 'cozy-device-helper'
-
-export const NATIVE_APP_INFOS = {
-  drive: {
-    appId: 'io.cozy.drive.mobile',
-    uri: 'cozydrive://',
-    name: 'Cozy Drive'
-  },
-  banks: {
-    appId: isAndroidApp() ? 'io.cozy.banks.mobile' : 'io.cozy.banks',
-    uri: 'cozybanks://',
-    name: 'Cozy Banks'
-  }
+import { UNIVERSALLINK_URL } from 'cozy-ui/transpiled/react/AppLinker/native.config'
+export const getUniversalLinkDomain = () => {
+  return UNIVERSALLINK_URL
```

Comment:
Is there a point to having a function and a constant exposing the same value?

Reply:
I prefer to have a getter. It gives us more flexibility to change the behavior / add check if needed (like checking if the url ends with a '/' or not).

Reply:
I don't mind the getter, but then wouldn't it be easier to not have the constant, or at least in the same file?

Reply:
We need to override the value :/